care/facility/tests/test_patient_api.py

```diff
@@ -510,7 +510,7 @@ def get_base_url(self) -> str:
         )
 
     def test_filter_by_admitted_to_bed(self):
-        self.client.force_authenticate(user=self.user)
+        self.client.force_authenticate(user=self.super_user)
```

Comment:
why is this failing for facility user? are permission not correct?

Reply:
@sainak perms are correct, the tests were failing because the old user did not have access to the facility (it was created by super_user)

Reply:
in that case you can set the users home facility while createing it `home_facility=cls.facility`